set.mm

```diff
@@ -421662,6 +421701,321 @@ corresponds to the size of the set of closed walks of length n (in an
       RXIXNVGVNVOVSVPVQVRTVTWAWBWC $.
   $}
 
+  ${
+    $d G g n v w $.
+    $( ` ( ClWWalksOnN `` G ) ` is an operator mapping a vertex ` X ` and a
+       nonnegative integer ` N ` to the set of closed walks on ` X ` of length
+       ` N ` in a graph ` G ` , as words over the set of vertices.
```

Comment:
X and N are called v and n in the statement.

Reply:
OK, I will replace the variables accordingly.

Reply:
... in my next PR!